teams/teams-ps/teams/New-CsTeamsVoiceApplicationsPolicy.md

```diff
@@ -9,42 +9,58 @@ ROBOTS: NOINDEX
 # New-CsTeamsVoiceApplicationsPolicy
 
 ## SYNOPSIS
+
 Creates a new Teams voice applications policy. TeamsVoiceApplications policy governs what permissions the supervisors/users have over auto attendants and call queues.
 
 ## SYNTAX
 
-```
+```ps
```

Comment:
For the Syntax section, I believe it should not have a language specifier (`ps`, in this case).